apps/namadillo/src/App/Common/Container.tsx

```diff
@@ -46,8 +46,9 @@ export const Container = ({
           className={clsx(
             "transition-transform duration-500 ease-out-expo",
             "pt-10 bg-black rounded-sm fixed top-0 z-[9999] w-[240px]",
-            "h-[calc(100svh-85px)] left-0 xl:z-0 xl:transition-none xl:pt-0 xl:w-auto xl:relative",
-            { "-translate-x-full xl:translate-x-0": !displayNavigation }
+            "h-[calc(100svh-90px)] left-[-240px] xl:left-0 xl:z-0",
```

Comment:
I see that we have a few errors here: 
- On mobile, the menu height should be the size of the screen, if we're testing in a vertical tablet, for example, we're gonna have some space below the panel
- I think it's better to keep the `-translate-x-full` instead of using `left-[-240px]`. If we wanna tweak the width in the future, we just need to change the `w-[240px]` .

Reply:
Example of the first item: 
<img width="574" alt="image" src="https://github.com/anoma/namada-interface/assets/810834/096dd11f-9635-4627-8455-5ccc2a4fee42">

Reply:
With the second point, the conditional `-translate-x-full` is what is causing the menu to quickly open and close when switching to the mobile view. I've added two new commits, one fixing the height and another to remove the duplication of the width variable. Could you have a look and let me know if you think this solution is OK or if you would prefer a different way?

Reply:
Now I see this behaviour! I don't think this is a problem, tbh, but can't see any other solution simpler than the one you've made!

Reply:
Ok, I think that makes sense so I've dropped that commit for now.